CMakeLists.txt

```diff
@@ -57,6 +57,11 @@ if(MSVC_GAPIR AND WIN32)
 endif()
 
 if(MSVC)
+    # turn off msvc specific warnings, define platform, turn on pdb generation
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4091 /wd4267 /wd4996 /wd4244 /D__x86_64 /Zi")
```

Comment:
Does the build fail if the warnings are not turned off? If so, please document for each warning type, why we can't fix the warnings, otherwise, leave the warnings on.